development-vm/README.md

```diff
@@ -1,276 +1,212 @@
-# Developing on GOV.UK Virtual Machines
+## Welcome to GOV.UK.
 
-Welcome to GOV.UK. The following instructions show you how to get your
-development environment running.
+Our development environment is an Ubuntu virtual machine (VM). We aim to achieve [dev-prod parity](http://www.12factor.net/dev-prod-parity).
 
-## 0. Context
+Follow the steps on this page to get your GDS development environment running with a [VirtualBox](https://www.virtualbox.org/) VM, managed and configured by [Vagrant](http://vagrantup.com/).
 
-Our development environment is an Ubuntu virtual machine with a view
-to achieving [dev-prod parity][1]. By default, the steps below will
-set you up with a [VirtualBox][2] VM, managed and configured by
-[Vagrant][3]. If you feel strongly about using another piece of
-software (such as VMWare) for your development VM, you may find
-instructions for doing so [on the wiki][4].
+> You'll need to use a Mac to follow this guide.
 
-Either way, you will need virtualisation enabled in your BIOS, otherwise it
-won't work. This tends to be enabled by default on Macs, but is worth
-checking for other manufacturers.
+It's expected to take roughly a day to complete this guide, start to finish. There's a lot of things to download, and loads of installers need to do their thing.
```

Comment:
It will take roughly one day to do everything in this guide from start to finish. There are lots of things to download, and loads of installers need to do their thing.